src/content/docs/cheerpj-jnlp-runner/04-guides/private-hosting-chrome.mdx

```diff
@@ -0,0 +1,127 @@
+---
+title: Private hosting (Chrome)
+description: Deploy the JNLP Runner chrome extension via Group Policy (Windows).
+---
+
+If your organisation requires to use our extension in multiple devices, we have amazing news for you! Our JNLP Runner extension is available for private hosting and deployment via Group Policy. This way, you can deploy our extension for multiple users under your organisation simultaneously and administrate it from a single place. This page will take you step by step on how to do this.
+
+Requirements:
+
+- The appropriate IT permissions at your organisation
+- The JNLP Runner extension CRX file.
+
+## The CRX File and Download URL
+
+The CRX file is a compressed format for extensions of browsers belonging to the Chromium family (such as Google Chrome). To obtain the CRX file of the JNLP runner extension for self-hosting, you will need to [contact us](https://cheerpj.com/contact/) and we will be happy to provide it for you alongside a license token.
+
+You will need to host the extension CRX file in the appropriate location of your IT infrastructure and create a download URL to access it. You will need this URL later in the tutorial.
+
+> [!info] About the Download URL
```

Comment:
```suggestion
> [!info] About `DownloadURL`
```
(This markdown might not get formatted, in which case I need to raise an issue in another project / go fix it but we can merge anyway)